mocks/api/v2/barong/resource/api_keys/GET.mock

```diff
@@ -2,6 +2,7 @@ HTTP/2 200
 access-control-allow-origin: *
 access-control-allow-methods: GET, POST, PUT, PATCH, DELETE
 access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept, Authorization
+Access-Control-Allow-Headers: *
```

Comment:
POST, PUT, PATCH, DELETE and TRACE verbs, being state changing verbs, should have a CSRF token attached to the request